src/wire/libevent_server.cpp

```diff
@@ -62,9 +68,19 @@ void Signal_Callback(UNUSED_ATTRIBUTE evutil_socket_t fd,
   event_base_loopexit(base, NULL);
 }
 
+void Status_Callback(UNUSED_ATTRIBUTE evutil_socket_t fd,
+                     UNUSED_ATTRIBUTE short what, void *arg) {
```

Comment:
In libevent, an event callback function must have such an interface so we cannot change it, though it may look a little redundant.